docsets/Framework7/docset.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Framework7",
+    "package": "f7",
+    "version": "2.3.1",
+    "archive": "versions/Framework7-2.3.1.tgz",
```

Comment:
This can't point to an archive inside `specific_versions`. Add it separately somewhere, even if it's a duplicate.

Reply:
Sorry again for that, it's now fixed